Gruntfile.js

```diff
@@ -10,26 +10,63 @@ var path = require('path');
 var e2e = require('./test/e2e/tools');
 
 var semver = require('semver');
-var fs = require('fs');
+var exec = require('shelljs').exec;
+var pkg = require(__dirname + '/package.json');
 
-var useNodeVersion = fs.readFileSync('.nvmrc', 'utf8');
-if (!semver.satisfies(process.version, useNodeVersion)) {
-  throw new Error('Invalid node version; please use node v' + useNodeVersion);
+// Node.js version checks
+if (!semver.satisfies(process.version, pkg.engines.node)) {
+  reportOrFail('Invalid node version (' + process.version + '). ' +
+               'Please use a version that satisfies ' + pkg.engines.node);
```

Comment:
You're too kind! If you feel strongly about it I will let you fix up the messaging in a future commit.